### ISSUE TITLE
Commissioner not sent user activation e-mail

### ISSUE THREAD

**Issue by rythos42** [TheNAF/naflm#133]
This one was weird. in CHOP, I'm a global administrator and Clark is our commissioner. One of our members signed up using the new registration process, but Clark's e-mail wasn't on the CC list when it should have been.

**Comment by rythos42** [comment]
Further problem I'm seeing now that the league-selection registration process is being used, is that Admins are being e-mailed as well as Commissioners. I'd prefer if Admins didn't even know anything was happening, so we don't get a lot of e-mails flying around.

**Comment by Shteve0** [comment]
Is this still open?

**Comment by rythos42** [comment]
Not really. My second comment is outstanding, about how I would like Admins not to be e-mailed this stuff. Could be a new Issue if you liked.

**Comment by Shteve0** [comment]
Cool, don't mind either way, just doing a quick scrape of the issues to see what's achievable for 1.1
